test/main.cpp

```diff
@@ -271,7 +274,7 @@ TEST_CASE("Manipulate cursor position through character writing",
             static_cast<uint8_t>(r2d2::claimed_display_cursor::OPEN_CURSOR));
 
         REQUIRE(cursor_post.cursor_x ==
-                (start_x + ((r2d2::display::st7735_128x160_s::width - start_x) / 8) * 8));//124
+                (start_x + ((r2d2::display::st7735_128x160_s::width - start_x) / 8) * 8));//due to int rounding in c++ this will result in 124
```

Comment:
waarom 124?

Reply:
omdat dat de waarde is waar de cursor uit moet komen